public/rss-news.xml.php

```diff
@@ -34,6 +34,9 @@
     $newsDate = date("D, d M Y H:i:s O", strtotime($news['Timestamp']));
     $newsImage = $news['Image'];
     $newsLink = config('app.url');
+    if (str_starts_with($news['Link'], config('app.url'))) {
+        $newsLink = $news['Link'];
+    }
```

Comment:
In _.env_, when I set `APP_URL` to "https://retroachievements.org" and then visit http://localhost:64000/rss-news, for some reason an exception is thrown:

> DOMDocument::createElement(): unterminated entity reference c=228801#228801

```php
    $article->appendChild($dom->createElement('link', $newsLink));
```

Reply:
I didn't have this issue, but I guess that it could come from some special characters in links.
I think that remediation for this issue would be treating them the same way as other text content appended in XML (which was lacking before).
Please, update the branch and check again, if issue still persists.

Reply:
That fixed the error 👍